src/user_management/user.graphql

```diff
@@ -18,7 +23,7 @@ input UserInput {
 extend type Query {
   users: [User!]!
   currentUser: User!
-  logInUser(email: String!, password: String!): User!
+  logInUser(email: String!, password: String!): AuthPayload!
```

Comment:
This change is not needed. You don't need to return the token via GraphQL. The server returns the token automatically via HTTP Headers. Don't forget to remove the `AuthPayload` type.